src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -158,8 +161,7 @@ public void deleteByUserIdAndTenantId(UUID consortiumId, String tenantId, UUID u
       .orElseThrow(() -> new ResourceNotFoundException(USER_ID + ", " + TENANT_ID, userId + ", " + tenantId));
 
     if (Boolean.TRUE.equals(userTenantEntity.getIsPrimary())) {
-      log.warn("Primary affiliation could not be deleted from API for user id: {} in the tenant: {}",
-        userId.toString(), userTenantEntity.getTenant().getId());
+      log.warn("Primary affiliation could not be deleted from API for user id: {} in the tenant: {}", userId.toString(), userTenantEntity.getTenant().getId());
```

Comment:
I don't think it makes sense to change formatting here and in similar places

Reply:
Reverted